python/HeaderService/hutils.py

```diff
@@ -218,33 +216,43 @@ def get_image_size_from_imageReadoutParameters(myData):
     return geom
 
 
-def start_web_server(dirname, port_number=8000, httpserver="http.server"):
+def start_web_server(dirname, port_number=8000, httpserver="http.server", logger=None):
+
+    if not logger:
+        LOGGER = create_logger()
+        LOGGER.setLevel(logging.INFO)
+        print("Creating new LOGGER")
+    else:
+        LOGGER = logger
```

Comment:
I strongly suggest you replace LOGGER with logger. Names in all caps names should be reserved for global constants (and variables, but don't use those!). It also allows you to simplify the code to the following:
```
if not logger:
    logger = create_logger()
    logger.setLevel(logging.INFO)
    self.log.info("Created a new logger")
```
where I also took the liberty of changing the print statement to a log statement.

Reply:
Will replace LOOGER by logger.
Thanks for catching the leftover `print` statement.